### ISSUE TITLE
Enhancement: Allow editor to be resized?

### ISSUE THREAD

**Issue by CheckeredFlag** [bizley/yii2-quill#26]
The docs state:

> **Editor box's height**
> Default editor height is 150px (this can be changed by setting 'options' parameter) and its box extends as new text lines are added.
> 

So I was able to increase the default editor size via:
```
'options' => [
    'style' => 'min-height: 300px;',
],
```
When you state "its box extends as new text lines are added", does this mean the editor's height should grow automatically?  It just scrolls for me.

It would be ideal if there was a way to make the editor resizable by dragging the bottom-right corner.

**Comment by bizley** [comment]
Hmm, is this maybe a problem with browser? See the below sample (Firefox, Linux):

![quill](https://user-images.githubusercontent.com/8577314/94522491-0b86c700-0230-11eb-83c8-d07a965be28b.gif)

As for the resize it's easy to do with css only with

```css
.ql-editor {
    resize: vertical;
    overflow-y: scroll;
}
```

although you need to override default `min-height` property then because the resize handler appears in the middle of the box instead of the bottom corner.

**Comment by CheckeredFlag** [comment]
Interesting.  I've found that it works just as you described when used as a standalone widget:
`<?= \bizley\quill\Quill::widget(['name' => 'editor', 'value' => '']) ?>`

However, it does not when attached to an ActiveField:
`<?= $form->field($model, $attribute)->widget(\bizley\quill\Quill::class, []) ?>`

The initial height shows only a single line (requiring `height` or `min-height`) and it won't grow dynamically.  Additionally, the css for a resize handle has no effect.

Is this a bug?  It obviously becomes much more complex under the hood when used in an ActiveForm.

**Comment by bizley** [comment]
Oh, I wasn't aware of that. Looks like a bug. I'll look at this tomorrow and get back.

**Comment by bizley** [comment]
Fixed in [3.2.0](https://github.com/bizley/yii2-quill/releases/tag/3.2.0)
